docs/content/v2024.2/yugabyte-platform/alerts-monitoring/anywhere-metrics.md

```diff
@@ -96,6 +96,21 @@ The following tables describe metrics available via the YugabyteDB Anywhere UI.
 | YCQL Op Latency (Avg) | The average time of DELETE, INSERT, SELECT, and UPDATE transactions, as well as other statements through the YCQL API. | An alert should be issued when latency is close to or higher than your application SLA. | ![YCQL Op Latency](/images/yp/metrics4.png) |
 | YCQL Op Latency (P99) | The average time of the top 99% of DELETE, INSERT, SELECT, and UPDATE transactions, as well as other statements through the YCQL API. | If this value is significantly higher than expected, then it might be a cause for concern and you might want to issue an alert.<br>You should check whether or not there are consistent spikes in latency. | ![YCQL Op Latency P99](/images/yp/metrics5.png) |
 
+### Per process
+
+Per process metrics provides details about the distribution of resources among YBA agents, and is currently available only from cloud providers (AWS, GCP, and Azure). The metrics presented are not intended for alert purposes, but they can be used when examining alerts on other metrics.
```

Comment:
currently available only for universes deployed on cloud providers?

Reply:
These metrics are also available for onprem universes, not only AWS,GCP, Azure. So in general it should be like "non-k8s universes"